.eslintrc.js

```diff
@@ -13,5 +13,11 @@ module.exports = {
     '@typescript-eslint/no-explicit-any': 0,
     '@typescript-eslint/explicit-module-boundary-types': 0,
     '@typescript-eslint/ban-ts-comment': 0,
+    '@typescript-eslint/no-namespace': [
+      2,
+      {
+        allowDeclarations: true,
```

Comment:
Enable `allowDeclarations` so it's possible to augment `window` object as a part of some tests. Those tests usually transfer the `worker` reference by setting it on the global `window` object, so some worker methods could be called from within the Puppeteer's environment during tests.